styles.css

```diff
@@ -0,0 +1,971 @@
+	font-size: 1rem;
+	font-weight: 700;
+	border-radius: 50%;
+	width: 1.5rem;
+	height: 1.5rem;
+}
+
+.alert-success,
+.alert-success button {
+	color: var(--color-success);
+	/* border: 1.5px solid var(--color-success); */
+	background-color: var(--color-success-bg);
+}
+.alert-warning,
+.alert-warning button {
+	color: var(--color-warning);
+	/* border: 1.5px solid var(--color-warning); */
+	background-color: var(--color-warning-bg);
+}
+.alert-danger,
+.alert-danger button {
+	color: var(--color-danger);
+	/* border: 1.5px solid var(--color-danger); */
+	background-color: var(--color-danger-bg);
+}
+
+.alert-primary,
+.alert-primary button {
+	color: var(--color-primary-dark);
+	/* border: 1.5px solid var(--color-primary); */
+	background-color: var(--color-primary-bg);
+}
+
+.alert-info,
+.alert-info button {
+	color: var(--color-secondary-dark);
+	/* border: 1.5px solid var(--color-primary); */
+	background-color: var(--color-secondary-bg);
+}
+
+/*Buttons*/
+
+.btn {
+	outline: none;
+	margin: 0.25rem;
+	padding: 0.5rem 1rem;
+	border: 1px solid transparent;
+	border-radius: 0.25rem;
+	color: white;
+	font-size: 1rem;
+	font-weight: 400;
+	cursor: pointer;
+	text-decoration: none;
+	transition: background-color 0.2s, color 0.2s;
+}
+
+.btn:hover {
+	box-shadow: inset 0 0 0 0.5px white;
+}
+
+.btn-primary {
+	background-color: var(--color-primary);
+}
+
+.btn-secondary {
+	background-color: var(--color-secondary);
+}
+
+.btn-success {
+	background-color: var(--color-success);
+}
+
+.btn-danger {
+	background-color: var(--color-danger);
+}
+
+.btn-warning {
+	background-color: var(--color-warning);
+}
+
+.btn-facebook {
+	background-color: var(--color-facebook);
+}
+
+.btn-twitter {
+	background-color: var(--color-twitter);
+}
+
+.btn-google {
+	background-color: var(--color-google);
+}
+
+/* Outline Buttons */
+
+.btn-outline-primary {
+	border-color: var(--color-primary);
+	color: var(--color-primary);
+	background-color: transparent;
+}
+
+.btn-outline-primary:hover {
+	background-color: var(--color-primary);
+	color: white;
+}
+
+.btn-outline-success {
+	border-color: var(--color-success);
+	color: var(--color-success);
+	background-color: transparent;
+}
+
+.btn-outline-success:hover {
+	background-color: var(--color-success);
+	color: white;
+}
+
+.btn-outline-danger {
+	border-color: var(--color-danger);
+	color: var(--color-danger);
+	background-color: transparent;
+}
+
+.btn-outline-danger:hover {
+	background-color: var(--color-danger);
+	color: white;
+}
+
+.btn-outline-warning {
+	border-color: var(--color-warning);
+	color: var(--color-warning);
+	background-color: transparent;
+}
+
+.btn-outline-warning:hover {
+	background-color: var(--color-warning);
+	color: white;
+}
+
+.btn-outline-info {
+	border-color: var(--color-secondary);
+	color: var(--color-secondary);
+	background-color: transparent;
+}
+
+.btn-outline-info:hover {
+	background-color: var(--color-secondary);
+	color: white;
+}
+
+/* Button Icon */
+
+i {
```

Comment:
If you are using font-awesome icons, give it a class name and then change its style otherwise it will target all icons with `i` tag.

Reply:
Done.